Expensify/expensify-common
# skip rendering invalid markdown

### Fixed Issues
$ https://github.com/Expensify/App/issues/17785

# Tests
1. Go to a chat and add a comment, `_*test_*`
2. Verify that text `test` is displayed in italic and is not formatted in bold style.

# QA
Same as test
